quantum/split_common/transport.c

```diff
@@ -161,12 +161,12 @@ bool transport_master(matrix_row_t master_matrix[], matrix_row_t slave_matrix[])
 #    endif
 
 #    if defined(LED_MATRIX_ENABLE) && defined(LED_MATRIX_SPLIT)
-    i2c_writeReg(SLAVE_I2C_ADDRESS, I2C_LED_MATRIX_START, (void *)led_matrix_eeconfig, sizeof(i2c_buffer->led_matrix), TIMEOUT);
+    i2c_writeReg(SLAVE_I2C_ADDRESS, I2C_LED_MATRIX_START, (void *)led_matrix_eeconfig.raw, sizeof(i2c_buffer->led_matrix.raw), TIMEOUT);
```

Comment:
Shouldn't it be this?
```suggestion
    i2c_writeReg(SLAVE_I2C_ADDRESS, I2C_LED_MATRIX_START, (void *)&led_matrix_eeconfig.raw, sizeof(i2c_buffer->led_matrix.raw), TIMEOUT);
```

Reply:
Probably

Reply:
Shouldn't the suspend bool have the same treatment, too?

Reply:
> 
> 
> Shouldn't the suspend bool have the same treatment, too?

Yes.

Reply:
Converting them all to pointers fixes the compilation errors, and removes the need to use `.raw`.  However, leaving the serial implementation with .raw.